core/domain/state_domain.py

```diff
@@ -772,65 +772,75 @@ def validate(self):
                 self.needs_update)
 
 
-class ContentTranslations(object):
+class WrittenTranslations(object):
     """Value object representing a content translations which stores
     translated contents of all state contents (like hints, feedback etc.) in
     different languages linked through their content_id.
     """
 
-    def __init__(self, content_translations):
-        """Initializes a ContentTranslations domain object."""
-        self.content_translations = content_translations
+    def __init__(self, translations_mapping):
+        """Initializes a WrittenTranslations domain object."""
+        self.translations_mapping = translations_mapping
 
     def to_dict(self):
```

Comment:
Make sure you have tests that validate that passing an object through to_dict and then from_dict result in the original object.

Reply:
Done!